crypto/bls/bls.go

```diff
@@ -0,0 +1,72 @@
+package bls
+
+import (
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/pairing/bn256"
+	"go.dedis.ch/kyber/v3/sign"
+	"go.dedis.ch/kyber/v3/sign/bdn"
+	"golang.org/x/xerrors"
+)
+
+const (
+	// PublicKeySize represents the length in bytes of a BLS public key.
+	PublicKeySize = 128
+
+	// SignatureSize represents the length in bytes of a BLS signature.
+	SignatureSize = 64
+
+	// PrivateKeySize represents the length in bytes of a BLS private key.
+	PrivateKeySize = 32
+)
+
+// AggregateSignatures aggregates multiple SignatureWithPublicKey objects into a single SignatureWithPublicKey.
+func AggregateSignatures(signaturesWithPublicKey ...SignatureWithPublicKey) (aggregatedSignature SignatureWithPublicKey, err error) {
```

Comment:
Because its more flexible. You can either call it with:

`AggregateSignatures(signature1, signature2)`

or if you have an array with

`AggregateSignatures(signatures...)`

Reply:
Cool!